src/config/transport_strategy.rs

```diff
@@ -6,3 +6,9 @@ pub struct TransportStrategy {
     /// Low-level gRPC settings for communicating with Momento.
     pub grpc_configuration: GrpcConfiguration,
 }
+
+impl TransportStrategy {
+    pub fn new(grpc_configuration: GrpcConfiguration) -> Self {
```

Comment:
same here, i'm thinking we don't want any `pub fn new` on any of these probably?